tests/build.gradle

```diff
@@ -16,6 +16,8 @@ tasks.withType(Test) {
 }
 
 task testLean(type: Test) {
+    exclude '**/*DockerExampleContainer*'
```

Comment:
can we not exclude these unit tests? or rather, why are you excluding them?
this is testing the skeleton image.